test/test_benchmark.py

```diff
@@ -0,0 +1,5 @@
+from preprocessor_benchmark import preprocessor_benchmark  
+import pytest
+
+def test_benchmark():
+    assert(preprocessor_benchmark() >= 1.22 and preprocessor_benchmark() <= 1.25)
```

Comment:
instead of this, just assert that the code returns a float value - we won't know for certain if the time is always between 1.22 and 1.25 seconds and wouldn't want to enforce this in our test (at least not now)